docs/Gemfile

```diff
@@ -23,3 +23,4 @@ gem "wdm", "~> 0.1.1", :platforms => [:mingw, :x64_mingw, :mswin]
 
 gem "just-the-docs"
 
+gem "webrick"
```

Comment:
Is this intentional?

Reply:
oops! yes, I need it locally to add to Ruby 3.0.0 since they removed it  but Jekyll needs it. I have been taking it out of commits until now but it was only a matter of time until I forgot. argh